### PR TITLE
update creatorkit documentation

### PR DESCRIPTION
**Pull request by mayank-sirotiya-imgtec**
Signed-off-by: Mayank Sirotiya <Mayank.Sirotiya@imgtec.com>

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Abhijit Mahajani <abhijit.mahajani@imgtec.com>
+Mayank Sirotiya <mayank.sirotiya@imgtec.com>
+Manohar Narkhede <manohar.markhede@imgtec.com>
```
Comment (abhijit-mahajani-imgtec) [inline CONTRIBUTORS]: spelling --> manohar.narkhede@imgtec.com

File: CONTRIBUTORS [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Abhijit Mahajani <abhijit.mahajani@imgtec.com>
+Mayank Sirotiya <mayank.sirotiya@imgtec.com>
+Manohar Narkhede <manohar.markhede@imgtec.com>
+Rahul daga <rahul.daga@imgtec.com>
+Prateek Prajapati <prateek.prajapati@imgtec.com>
```
Comment (abhijit-mahajani-imgtec) [inline CONTRIBUTORS]: Pratik.Prajapati

File: CodingGuidelines.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+            else
+            {
+                LOG(LOG_ERR, "Failed to login as device");
+            }
+        }
+    }
+    return false;
+}
+```
+**Note:**
```
Comment (nikhil-zinjurde-imgtec) [inline CodingGuidelines.md]: Better to have "General guidelines" or "Spacing guidelines" as title here

File: CodingGuidelines.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+ * @param *session holds client session.
+ * @return true if object is already defined, else false.
+ */
+bool IsLedObjectDefined(const AwaClientSession *session)
+{
+....
+}
+```
+
+#### Good coding practise
```
Comment (nikhil-zinjurde-imgtec) [inline CodingGuidelines.md]: Spelling mistake -> practice

File: CodingGuidelines.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+* Ideally functions should have a single exit point. The use of multiple return points is dangerous and makes it harder to read the code. Always consider what will happen if someone needs to append code to your function.
+* Where possible definitions should be used rather than raw values, e.g. use *AllocMessageBuf(MAX_MESSAGE_SIZE)* rather than  *AllocMessagBuf(1024)*. This ensures that all code sharing the same defined value will still work if the 'magic' value is updated.
```
Comment (nikhil-zinjurde-imgtec) [inline CodingGuidelines.md]: "Wherever possible, "

**Comment by abhijit-mahajani-imgtec** [comment]
Approved.

**Pull request merged**